react/src/components/molecules/MainNav/index.js

```diff
@@ -110,16 +110,31 @@ class MainNav extends Component {
       <div className="ma__main-nav">
         <ul className="ma__main-nav__items" role="menubar">
           {mainNav.map((item, index) => {
+            const covidTopLink = item.text.toLowerCase().includes('covid') || false;
             const topItemClasses = classNames({
               'ma__main-nav__item': true,
               'is-active': item.active,
-              'has-subnav': item.subNav
+              'has-subnav': item.subNav && !covidTopLink
             });
             const buttonId = `button${index}`;
             const liId = `li${index}`;
             const isExpanded = navSelected === liId;
             const itemBody = [];
-            if (item.subNav) {
+            if (covidTopLink) {
+              const covidTopLinkProps = {
+                id: buttonId,
+                index,
+                className: classNames([
+                  'ma__main-nav__top-link',
+                  'cv-alternate-style'
+                ]),
+                role: 'menuitem',
+                'aria-label': (isExpanded) ? `Hide submenu for ${item.text}` : `Show submenu for ${item.text}`,
```

Comment:
@tom-fleming This link has text label, so aria-label is not necessary. Plus, it has nothing to show/hide.  Can. you remove this line?

Reply:
@ygannett Just pushed a fix for this. Please have another look